src/sas/qtgui/Perspectives/Inversion/InversionPerspective.py

```diff
@@ -435,3 +435,10 @@ def updateFromParameters(self, params):
         inversion_widget = self.currentWidget()
         if isinstance(inversion_widget, InversionWidget):
             inversion_widget.updateFromParameters(params)
+
+    def reset(self):
+        """
+        Reset the Inversion perspective to an empty state
+        """
+        self.resetTab(self.currentIndex())
```

Comment:
This only clears the active P(r) tab. All other tabs retain their, now orphaned, data.